docs/guide/website/versioned_docs/version-11.7.3/build/nlu.md

```diff
@@ -91,7 +91,7 @@ You can use that metadata in your flows to create transitions when a specific in
 
 To enable debugging of the NLU module, make sure that `debugModeEnabled` is set to `true` in your `data/global/config/nlu.json` file.
 
-> **Tip**: In production, you can also use the `BP_NLU_DEBUGMODEENABLED` environement variable instead of modifying the configuration directly.
+> **Tip**: In production, you can also use the `BP_NLU_DEBUGMODEENABLED` enviroement variable instead of modifying the configuration directly.
```

Comment:
```suggestion
> **Tip**: In production, you can also use the `BP_NLU_DEBUGMODEENABLED` environment variable instead of modifying the configuration directly.
```